### ISSUE TITLE
Code block direction in RTL interface

### ISSUE THREAD

**Issue by merrime-n** [BookStackApp/BookStack#4809]
### Describe the Bug

When we create a code block in a RTL page and modify its direction to LTR, it looks flawless in the editing mode. But when we save the document, it goes RTL and is not properly readable.

### Steps to Reproduce

1. Go to an RTL interface.
2. Create a code block in a page.
3. In edit mode, make it left to right.
4. Save the page.
5. See? It is right to left.

### Expected Behaviour

The code is expected to be rendered left to right regardless of the interface language.

### Screenshots or Additional Context

![Capture](https://github.com/BookStackApp/BookStack/assets/100111649/47b81456-a2c7-4113-86a5-c3eb2ceb7352)

### Browser Details

Google Chrome Version 104.0.5112.102 (Official Build) (64-bit) On Windows 10

### Exact BookStack Version

BookStack v23.12

**Comment by ssddanbrown** [comment]
Thanks for reporting @okaeiz, I've assigned this to be fixed in the next feature release.
If a temporary workaround was desired, you can add this below code to the "Custom HTML Head Content" customization setting to force all code blocks to ltr:

```html
<style>
  .cm-editor, .page-content pre {direction: ltr;}
</style>
```

**Comment by ssddanbrown** [comment]
This has now been addressed within commit 999d41a7f5455f4c7f9f175b5718d82ee529df2b.
This will be part of the next feature release.
